__tests__/nc-news.test.js

```diff
@@ -48,6 +48,20 @@ describe('GET /api/articles/:article_id', () => {
 
     expect(body.error).toBe('Article not found');
   });
+
+  it('400: bad request when id is not a integer', async () => {
+    const {body} = await request(app).get('/api/articles/badId').expect(400);
+
+    expect(body.error).toBe('Bad request');
+  });
+});
+
+describe('GET /api/articles/:article_id', () => {
+  it('200: returns the comment count in relation to the requested article ', async () => {
```

Comment:
You've now got 2 describe block with the same description. This test can just go into the previous describe.